repo/packages/M/marathon-lb/19/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "2.0",
+  "name": "marathon-lb",
+  "version": "1.7.0",
+  "scm": "https://github.com/mesosphere/marathon-lb",
+  "description": "HAProxy configured using Marathon state",
+  "maintainer": "support@mesosphere.io",
+  "tags": ["loadbalancer", "service-discovery", "reverse-proxy", "proxy", "haproxy"],
+  "preInstallNotes": "We recommend at least 2 CPUs and 1GiB of RAM for each Marathon-LB instance. \n\n*NOTE*: ```Enterprise Edition``` DC/OS requires setting up the Service Account in all security modes. \nFollow these instructions to setup a Service Account: https://docs.mesosphere.com/administration/id-and-access-mgt/service-auth/mlb-auth/",
```

Comment:
I understand you are making a simple version update.   Making a minor non-blocking comment here.   It probably makes sense to mention in the pre install notes that the 2 CPU, 1G recommendation is on the public node and that a public node must be available by default. (but perhaps no one reads these pre-install notes:)